nodeup/pkg/model/kubelet.go

```diff
@@ -105,6 +113,37 @@ func (b *KubeletBuilder) Build(c *fi.ModelBuilderContext) error {
 
 	c.AddTask(b.buildSystemdService())
 
+	if b.Distribution == distros.DistributionCoreOS {
+		sha1URL := os.Getenv("SOCAT_SHA1_URL")
```

Comment:
It's a SHA256, should probably change that (realizing that means you'll have to change it in the GH release too)